features.txt

```diff
@@ -1231,6 +1238,13 @@ balance.ender_dragon_always_spawn_egg
 	desc:
 		Ender dragon always spawn a dragon egg when killed.
 
+balance.use_items_while_riding
```

Comment:
Balance feels like a weird place for something this minor, but you're correct in thinking it's a bit much for Tweaks. Weird Tweaks is probably the best fit.